test/chainsaw/non-blocking-deployment/keptnconfig-blocking.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: options.keptn.sh/v1alpha1
 kind: KeptnConfig
 metadata:
-  name: "some-config"
+  name: non-blocking-config
 spec:
-  OTelCollectorUrl: "some-url"
```

Comment:
to really preserve the test behavior, I would keep it. This way we can verify that `OTelCollectorUrl` is a valid property for the spec

Reply:
If we keep this parameter pointing to a non-existing url, we will get multiple errors in the lifecycle-operator logs that the OTel URL is invalid (the operator will try to set it up and connect to it). That will polish the logs and it will have no value for us except that the parameter can be set.

Reply:
maybe we can add a proper test for this as part of #3121

Reply:
you mean something like an integration test for the OTel Collector?

Reply:
adapted the ticket to include an integration test to check the OTelCollectorUrl, #3121